<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C3LAP002" ignored="3" name="WE-CompanyContacts" tests="1" failures="1" timestamp="23 Aug 2019 07:32:18 GMT" time="78.354" errors="0">
  <testcase name="we_CompanyContacts" time="62.2" classname="com.mobileautomation.tests.WE_CommonSuite">
    <failure type="java.lang.AssertionError" message="this will make the test method fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: this will make the test method fail expected [true] but found [false]
	at commonUtil.WebDriverUtils.takeScreenshot(WebDriverUtils.java:60)
	at com.mobileautomation.tests.WE_CommonSuite.we_CompanyContacts(WE_CommonSuite.java:116)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- we_CompanyContacts -->
  <testcase name="we_Notification" time="0.0" classname="com.mobileautomation.tests.WE_CommonSuite">
    <ignored/>
  </testcase> <!-- we_Notification -->
  <testcase name="weConnect_GenericFunctionality" time="0.0" classname="com.mobileautomation.tests.WE_CommonSuite">
    <ignored/>
  </testcase> <!-- weConnect_GenericFunctionality -->
  <testcase name="weConnect_NetworkStatus" time="0.0" classname="com.mobileautomation.tests.WE_CommonSuite">
    <ignored/>
  </testcase> <!-- weConnect_NetworkStatus -->
</testsuite> <!-- WE-CompanyContacts -->
